<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>McDonald's</string>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Pedidos</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Clientes</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetClient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>421</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>137</number>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sexo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPF</string>
       </property>
      </column>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>443</x>
        <y>60</y>
        <width>16</width>
        <height>381</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="button_FetchClient">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ClientPhoto">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>141</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>Foto do Cliente</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_NewClient">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo Registro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_DeleteClient">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NameClient">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>245</y>
        <width>47</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NameClient">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>240</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Client">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Digite o nome do cliente</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SexClient">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>350</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Sexo:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_M">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>350</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Masculino</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_F">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>350</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Feminino</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NameClient_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>295</y>
        <width>47</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CPFClient">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ex.: 123.456.789-00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_Picture">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>190</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Procurar...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_Save">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>410</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_Cancel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>594</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_Client">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>421</width>
        <height>23</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Funcion√°rios</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Lanches</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Bebidas</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Combos</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Sobre</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>mcdonalds/images/mc_ban.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>mcdonalds/images/bottom.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_Client</tabstop>
  <tabstop>tableWidgetClient</tabstop>
  <tabstop>button_FetchClient</tabstop>
  <tabstop>button_NewClient</tabstop>
  <tabstop>button_DeleteClient</tabstop>
  <tabstop>button_Picture</tabstop>
  <tabstop>lineEdit_NameClient</tabstop>
  <tabstop>lineEdit_CPFClient</tabstop>
  <tabstop>radioButton_M</tabstop>
  <tabstop>radioButton_F</tabstop>
  <tabstop>button_Cancel</tabstop>
  <tabstop>button_Save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
